src/app/learn/overview/index.html

```diff
@@ -20,6 +20,9 @@
       SKY UX enables developers to focus on features instead of infrastructure. It creates user experience interactions and allows feature engineers to consume them.
     </p>
   -->
+  <sky-alert alertType="info">
+    This site describes the version of SKY UX that supports versions of Angular after the AngularJS (1.x) framework. For information about the version of SKY UX that supported the AngularJS (1.x) framework, visit <a href="https://skyux.developer.blackbaud.com/">skyux.developer.blackbaud.com/</a>.
```

Comment:
Could it be simplified a bit? Something like, "This site describes the version of SKY UX that supports the latest version of Angular. For information about the version of SKY UX that supports the AngularJS framework, visit..."

Reply:
Sounds good to me. I wasn't sure if "the latest version of Angular" was correct, which is why I had the longer, more unwieldy wording in place before. But if this shorter version is accurate, I'm all for it. I've updated the text with our suggested version. Thanks!